gradle/libs.versions.toml

```diff
@@ -1,34 +1,50 @@
 [versions]
 # Project
-android_gradle_plugin = "8.1.4"
-kotlin = "1.9.20"
-ksp = "1.9.20-1.0.14"
+android_gradle_plugin = "8.2.2"
+kotlin = "1.9.22"
+ksp = "1.9.22-1.0.17"
+
+# AndroidX
+activity = "1.8.2"
 
 # Compose
-compose_bom = "2023.10.01"
-composecompiler = "1.5.4"
+compose_bom = "2024.02.00"
+composecompiler = "1.5.9"
 
 # Accompanist
-accompanist = "0.32.0"
+accompanist = "0.34.0"
 
 # Others
 showkase = "1.0.2"
-ui-tooling-preview-android = "1.5.4"
+ui-tooling-preview-android = "1.6.1"
+kover = "0.7.5"
+roborazzi = "1.9.0"
```

Comment:
We will probably need to setup Renovate on this repo to handle the upgrades

Reply:
Let me know if it's something I can do.

Reply:
I take the point.

Reply:
Done in https://github.com/element-hq/compound-android/pull/7